presto-main/src/test/java/io/prestosql/operator/TestPartitionedOutputOperator.java

```diff
@@ -59,12 +59,13 @@
 
 public class TestPartitionedOutputOperator
 {
-    private static final int PAGE_COUNT = 100;
-    private static final int POSITIONS_PER_PAGE = 1000;
+    private static final DataSize MAX_MEMORY = new DataSize(50, MEGABYTE);
+    private static final DataSize PARTITION_MAX_MEMORY = new DataSize(5, MEGABYTE);
 
+    private static final int PAGE_COUNT = 10;
+    private static final int POSITIONS_PER_PAGE = 1000;
     private static final int PARTITION_COUNT = 512;
```

Comment:
Maybe there was some hidden reason why it was using so high values? Can we ask the author or someone with more context?